### PR TITLE
apicurio registry to be installed in each student namespace.

### PR DESCRIPTION
**Pull request by varodrig**
SUMMARY:

This is a workload  to install Apicurio Registry Operator at namespace level per each student. In this case at the schema-registry-userX. For now it'll be used in the Adv. CND VILT, but can be used by any catalog.

ISSUE TYPE

New config Pull Request

COMPONENT NAME
ilt-adv-cloud-native-development-prod

### CODE REVIEW

File: ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/remove_workload.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+- name: Remove OpenShift Apicurio Project
+  k8s:
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/remove_workload.yml:3]: Does this need a when clause to only run when "set up users" is set to true?

File: ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    name: install_operator
+  vars:
+    install_operator_action: install
+    install_operator_name: apicurio-registry-operator
+    install_operator_namespace: 'user{{ item }}-schema-registry'
+    install_operator_channel: "{{ ocp4_workload_apicurio-registry_operator_channel }}"
+    install_operator_catalog: ""
+    install_operator_automatic_install_plan_approval: "{{ ocp4_workload_apicurio-registry_operator_automatic_install_plan_approval | default(true) }}"
+    install_operator_starting_csv: "{{ ocp4_workload_apicurio-registry_operator_starting_csv }}"
+    install_operator_use_catalog_snapshot: true
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml:23]: This should probably come from a variable and not be hardcoded to true.

File: ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+- name: Create OpenShift Apicurio Project
+  k8s:
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml:3]: Is this related to the variable to "set up users" ? If so it probably needs a when clause.

**Review by wkulhanek — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml (hunk 3 above)
Comment (varodrig) [reply ansible/roles_ocp_workloads/ocp4_workload_apicurio_registry_operator /tasks/workload.yml:3]: I'm not setting up users. I'm creating a project per users.

**Review by wkulhanek — APPROVED** [review]
